### DISCUSSION TITLE
Let PgUp / PgDown key able to select the song instead of only scroll the list

### DISCUSSION THREAD

**Discussion by MeerukYang** [ppy/osu#30780] [Ideas]
PgUp / PgDown key at menu would now only scroll the list.

https://github.com/user-attachments/assets/da8eb912-417e-4be8-940b-9aec779726ed

If using ↑ / ↓ key, it unfold the songs, thus you have to tap repeatly. if let PgUp / PgDown key able to select the songs, it would be much better. Just like selecting previous / next group, but more songs would be skipped.

**Comment by peppy** [comment]
Maybe when holding shift or something? I'm not sure they should select by default (although arguably that would match how pgup/pgdown work in text editors on most systems)..
